server/events/command_runner.go

```diff
@@ -146,9 +149,15 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(baseRepo models.Repo, headRepo
 
 	log := c.buildLogger(baseRepo.FullName, pull.Num)
 	defer c.logPanics(baseRepo, pull.Num, log)
+
+	scope := c.StatsScope.Scope("autoplan")
```

Comment:
Is there a reason why this is not in a decorator?

Reply:
I need to add it to `CommandContext` in order to preserve the scope and I don't want to change the interface to pass it around. 

Since this is basically the entrypoint for commands, it seemed fine to me to just modify this code.